db/seeds.rb

```diff
@@ -19,9 +19,43 @@
 Packaging.create(name: 'Paper bag')
 Packaging.create(name: 'Recyclable')
 
-Certification.create(name: 'Halal')
+@halal = Certification.create(name: 'Halal')
 Certification.create(name: 'Kosher')
 Certification.create(name: 'Non-GMO')
 
 Nutrition.create(name: 'Superfood')
 Nutrition.create(name: 'High in protein')
+
+@vendor_picture = "http://image.shutterstock.com/image-vector/vector-illustration-supermarket-grocery-store-260nw-518613133.jpg"
+
+@first_vendor = Vendor.create(name: 'First Vendor', picture: @vendor_picture, address: "", facebook: "", twitter: "", instagram: "", story: "", mission: "")
```

Comment:
We should create actual data to fill this, instead of leaving the fields empty; Adnan wanted the data so he could see offline, and we should have data to fill the details pages too. 

Maybe grab the same data that’s currently deployed from the Heroku version?

Reply:
Added Some Data.